themes-default/slim/views/addShows_addExistingShow.mako

```diff
@@ -184,7 +170,7 @@ const startVue = () => {
                         <li><app-link href="addShows/existingShows/#tabs-2">Customize Options</app-link></li>
                     </ul>
                     <div id="tabs-1" class="existingtabs">
-                        <%include file="/inc_rootDirs.mako"/>
+                        <root-dirs @update:root-dirs-value="rootDirsUpdated"></root-dirs>
```

Comment:
You should look into https://vuejs.org/v2/guide/components-custom-events.html#Binding-Native-Events-to-Components since this can be treated like an input field.

Reply:
I'm only watching for changes in value, so it doesn't refresh the directory list if you change the selection in the root-dirs component - only if you change the default root-dir, or add/edit/delete a root dir.
As far as I could tell, there is no native event for that (you can't reliably detect when an option was edited, added or removed from a select?

Reply:
I can see how this can simplify the root-dirs component if I bind a `change` event handler directly on the `<select>` element. I can do that, and looks like it is working.

However I still need a watcher for the "root-dirs-value", which is basically this: `0|C:\TV|D:\TV`

Reply:
So we'd need to provide two seperate native elements, a select and a input box?

Reply:
An input box? Why?

Reply:
As I said on Slack,
I'm not planning on keeping the `#rootDirText` input element.
It will be removed when `addShowOptions` is converted to Vue, so I don't see a reason to add a native event watcher on it.

But I did add `v-bind="$attrs" v-on="$listeners"`. Not sure if we're going to use it.